Task/services/users/service.go

```diff
@@ -0,0 +1,87 @@
+package service
+
+import (
+	"errors"
+	"zopsmart/Task/models"
+	service "zopsmart/Task/services"
+	store "zopsmart/Task/stores"
+)
+
+type UserServiceHandler struct {
+	stores store.Store
+}
+
+func New(store store.Store) service.Services {
+	return UserServiceHandler{stores: store}
+}
+
+func (u UserServiceHandler) GetAllUsersService() ([]models.User,error){
+	return u.stores.GetAllUsersStore()
+}
+
+func (u UserServiceHandler) GetUserByIdService(Id int) (models.User, error) {
+	ok := validateId(Id)
+	if !ok{
+		return models.User{},errors.New("invalid Id")
+	}
+	res, err := u.stores.GetUserById(Id)
+	if err != nil {
+		return res, errors.New("error occured while getting the data")
```

Comment:
return same err that we getting from store layer, don't need to create one., this will apply in all cases.